### DISCUSSION TITLE
Implement VS Code Web app here

### DISCUSSION THREAD

**Discussion by TechyAditya** [DustinBrett/daedalOS#57] [Ideas]
VS code is already usable online (vscode.dev). It's good to see the modified version optimised for windowed mode in this OS.

**Comment by DustinBrett** [comment]
Thanks for the suggestion @TechyAditya. I will consider adding a bookmark to that url in my Browser app. My first is purposely slim down and I think of it more like if VS Code and Notepad had a baby.

**Reply by a-reisberg** [reply]
Actually, it would be great to be able to run https://github.com/coder/code-server and have a full-fledge dev experience. Is this even possible?

**Reply by DustinBrett** [reply]
Thanks for the suggestion but I don't think it would be possible this way if I wanted it to remain client side only, which I do. I do have plans to work more on Monaco IDE in the near future, but full VS Code without a server I am unsure when that would be possible.

**Reply by a-reisberg** [reply]
I see. Thank you!

**Reply by DustinBrett** [reply]
I will keep an eye on this btw. I do want to have a cool IDE like VS Code. But I also want to remain client side as much as possible. I think things are moving in that direction.

**Reply by a-reisberg** [reply]
Awesome! I was hoping that the whole code-server can be compiled to wasm and run in a browser. But maybe that's too much work :).

**Reply by DustinBrett** [reply]
Haha ya that might be a while away. But the VSCode does indeed use Monaco so I am on my way. It also uses xterm.js which is what I use for Terminal. I've also integrated Prettier code formatting for some languages. Step by step I hope to build up the features. I doubt it will have extension support though tbh.

**Reply by a-reisberg** [reply]
I see. [daedalOS](https://github.com/DustinBrett/daedalOS) as a whole is very impressive. I looking forward to seeing more cool stuff done in it!
